Entities/Characters/Builder/BuilderLogic.as

```diff
@@ -648,3 +648,30 @@ void onAddToInventory(CBlob@ this, CBlob@ blob)
 		SetHelp(this, "help inventory", "builder", "$Help_Block1$$Swap$$Help_Block2$           $KEY_HOLD$$KEY_F$", "", 3);
 	}
 }
+
+void onAttach(CBlob@ this, CBlob@ attached, AttachmentPoint @attachedPoint)
+{
+	if (attached is null) return;
+	if (attached.getName().getHash() == 1346747564) // (attached.getName() == "seed")
```

Comment:
There is no need to optimise this by comparing the hash. Just compare the name and combine the two `if` statements into one.